app/knights/inventory.py

```diff
@@ -0,0 +1,27 @@
+from app.items.weapon import Weapon
+from app.items.armour import Armour
+from app.items.potion import Potion
+
+
+class Inventory:
+    def __init__(self,
+                 weapon: dict,
+                 armours: list[dict],
+                 potion: dict
+                 ) -> None:
+        if weapon:
+            self.weapon = Weapon(weapon["name"], weapon["power"])
+        else:
+            self.weapon = None
+
+        if armours:
+            self.armour = [
+                Armour(arm["part"],
+                       arm["protection"]) for arm in armours]
+        else:
+            self.armour = []
+
+        if potion:
+            self.potion = Potion(potion["name"], potion["effect"])
+        else:
+            self.potion = None

```

Comment:
try to write these more simply, create a method for this or another solution, it looks worse